thesis-fe/src/components/pages/Profile_page.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Navigation from '../container/Navigation';
+import styled from 'react-emotion';
+
+
+
+const Container = styled('div')`
+  border-radius: 200px;
+  width: 100vw;
+  height: 100vh;
+  margin-right: 2rem;
+`;
+
+
+const ContainerOptions = styled('div')`
+  border-radius: 200px;
+  width: 100vw;
+  height: 90vh;
+  margin-right: 2rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  padding: 3rem;
+  align-items: center;
+`;
+
+const Profile_page = () => {
+  const logout = () => {
+    localStorage.setItem('logged','');
+    localStorage.setItem('id','');
+    window.location.replace('/auth');
+  }
+
+  return (
+    <Container>
+      <Navigation />
+      <ContainerOptions>
+        <h1 onClick={logout}>Logout</h1>
+        <h1>Change avatar</h1>
+        <h1>Send us a message</h1>
```

Comment:
Don't overuse `h1`. Actually, MDN recommends avoiding using it more than once in a page, once per section to be more correct.

Also, it seems that you need something else, like links or buttons.